.gitattributes

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
```

Comment:
Does this need to be here?

Reply:
It’s probably there because I copypasted it from a .py file. I’ll remove it